cnd/src/network/comit_ln.rs

```diff
@@ -106,10 +106,18 @@ impl ComitLN {
         id: LocalSwapId,
         create_swap_params: HanEtherereumHalightBitcoinCreateSwapParams,
     ) {
-        let digest = create_swap_params.clone().digest();
-
+        for (_, params) in self.swaps.iter() {
+            if *params == create_swap_params {
+                tracing::warn!(
+                    "swap with same parameters already exists, aborting swap initiation"
+                );
+                return;
```

Comment:
I would like to have error handling for this. Up till here, we are still in the context of the HTTP request, so we can actually return an error to the user right away.

Reply:
Cool, will do.